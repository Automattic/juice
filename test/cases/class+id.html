<html><body>
<p id="test" class="woot">Woot</p>
</body></html>
