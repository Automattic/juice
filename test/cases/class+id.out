<html><body>
<p id="test" class="woot" style="border: 1px solid #f00; display: inline-block;">Woot</p>
</body></html>
